src/reducer/database/osm/models.ts

```diff
@@ -0,0 +1,1085 @@
+//#region > Imports
+
+//> Reconstructor
+// SOAssambler for SO objects
+import { SOAssambler } from "./reconstructor";
```

Comment:
This whole file needs an upgrade.

1. We allow up to 119 characters as this is the width of GitHub code review, anything longer requires horizontal scrolling which makes review more difficult. Documentation, comments, and docstrings should be wrapped at 79 characters.

2. There are some problems. xD